addons/account/models/account_move.py

```diff
@@ -4071,13 +4071,12 @@ def _get_all_reconciled_invoice_partials(self):
                 part.debit_move_id AS counterpart_line_id
             FROM account_partial_reconcile part
             WHERE part.credit_move_id IN %s
-        '''
-        self._cr.execute(query, [tuple(reconciled_lines.ids)] * 2)
+        ''', *params)
```

Comment:
The code doesn't need to be smart.  Go for the plain stupid way:
```suggestion
        ''', reconciled_lines._ids, reconciled_lines._ids)
```